presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSessionProperties.java

```diff
@@ -78,6 +78,7 @@ public final class IcebergSessionProperties
     private static final String NESSIE_REFERENCE_HASH = "nessie_reference_hash";
     public static final String READ_MASKED_VALUE_ENABLED = "read_null_masked_parquet_encrypted_value_enabled";
     public static final String PARQUET_DEREFERENCE_PUSHDOWN_ENABLED = "parquet_dereference_pushdown_enabled";
+    public static final String WORKER_TYPE = "worker_type";
```

Comment:
Use PUSHDOWN_FILTER_ENABLED, similar to Hive.

public static final String PUSHDOWN_FILTER_ENABLED = "pushdown_filter_enabled";

Reply:
Done, as requested by you.